converters/fromZigbee.js

```diff
@@ -716,6 +716,23 @@ const converters = {
             };
         },
     },
+    ias_keypad: {
+        cluster: 'ssIasZone',
+        type: 'commandStatusChangeNotification',
+        convert: (model, msg, publish, options, meta) => {
+            const zoneStatus = msg.data.zonestatus;
+            return {
+                alarm_1: (zoneStatus & 1) > 0,
```

Comment:
Are all attributes used by the keypad?

Reply:
I can't find any good documentation on this device, but tamper and restore_reports are reported by the keypad, not sure about battery_low. The others are probably not used.

Reply:
![image](https://user-images.githubusercontent.com/6302887/135914524-6cb05d39-2441-4c89-b8c3-d7825c1023cf.png)

Reply:
Please remove the ones not used by the device (with that I mean those which never change)